gluoncv/model_zoo/pose_resnet.py

```diff
@@ -4,34 +4,33 @@
 # Written by Bin Xiao (Bin.Xiao@microsoft.com)
 # ------------------------------------------------------------------------------
 
-from __future__ import division
+# coding: utf-8
+# pylint: disable=missing-docstring
 
-__all__ = ['get_pose_resnet']
+from __future__ import division
 
-import torch
-import torch.nn as nn
+__all__ = ['get_pose_resnet', 'pose_resnet18', 'pose_resnet34',
+           'pose_resnet50', 'pose_resnet101', 'pose_resnet152']
 
 from mxnet.context import cpu
 from mxnet.gluon.block import HybridBlock
 from mxnet.gluon import nn
 
-def _conv3x3(out_planes, stride=1):
-    return nn.Conv2D(out_planes, kernel_size=3, stride=stride,
-                     padding=1, bias=False)
-
 class BasicBlock(HybridBlock):
```

Comment:
would our basic resnet definition work here? Hopefully we can reuse our ResNet-A to D models